app/views/categories/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class ="c-index-cont">
+    <div>
+        <%= render "shared/navbar" %>
+        <%= render "shared/messages" %>
+        
+            <% @categories.each do |category| %>
+                <div class="c-index-cont__category__name">
+                <%= link_to category_path(category), class: "c-index-cont__category" do %>
+                        <div class="img-name">
+                        <%= image_tag(category.icon_url, class: "c-index-cont__category__name__icon") %>
+                        <p class="c-index-cont__category__name__text"><%= category.name %></p>
+                        </div>
+                        <div class="amt">                    
```

Comment:
- [OPTIONAL] The card should have a category image, name, `created date`, and amount. kindly let display the creation date for each category card. 🛀🏻 

Your Implementation | Original Design
:---:|:---:
![image](https://github.com/JacintaNgwu/Budget-App/assets/79058364/a77c9b5c-5c87-4ee3-aa50-e5ca63311cad)| ![image](https://user-images.githubusercontent.com/79058364/219786288-a43c8eb4-d309-4b09-9b4e-02b929ccd7fb.png)